### DISCUSSION TITLE
proposal(localization): add Chinese version of docs into kiali.io

### DISCUSSION THREAD

**Discussion by Xunzhuo** [kiali/kiali#4612] [General]
Hi communiy :),
I wonder if we can add Chinese version of docs into kiali.io?
Because I think a large number of developers in China are interested in kiali, who are trying to use kiali into production.
If you folks think it is OK to do this, I will lead the Chinese localization job from my experience as the istio WG doc maintainer and the member of Kubernetes Docs. :)
Looking forward to your advice~

**Comment by lucasponce** [comment]
Hi @Xunzhuo,
Thanks for starting your discussion.
Main limitation is the knowledge of the language and how that can impact to keep documentation updated across changes.

But if that may help to grow a community specially in the Chinese community, I'm +1 for it.

Perhaps at the beginning Chinese doc should have some disclaimer that some parts could be up-to-date until someone could translate, I'm not familiar how that works in other projects.

**Reply by Xunzhuo** [reply]
Hi @lucasponce, thanks for your advice :)
For the limitation of the language, I think it is not a big problem as I am from China. I can do the localization job at the beginning.
My plan is below:
+ Translate the latest version like `1.44` of kiali myself in my free time.
+ After finishing this, I will encourage Chinese developers to participate in localization jobs including:
   + Track the updated/added/deleted english docs and keep Chinese docs update. 
   + Of course, I will keep tracking changes in english docs and keep updating Chinese Docs.

I think the localization job can help to grow kiali community and let more people to learn, use, contribute to kiali community in China. If you folks think it is OK, I will start to do this :)

**Reply by lucasponce** [reply]
Thanks @Xunzhuo for the interest.
Let's see if more maintainers support this idea and then we can plan how it could be planned.
From my side, I really appreaciate that you guys are interested on helping on this task.

**Comment by jshaughn** [comment]
The timing for this proposal is good.  We have recently completed a large revision of the website, including a change to the Docsy theme.  With Docsy, there is some built-in support for localization.   The effort would have to be 99.9% contribution, the current maintainers would only be able to offer some guidance on the site structure itself.

Note that there is no intention to localize Kiali itself.

**Comment by lucasponce** [comment]
> Note that there is no intention to localize Kiali itself.

"Today". But if there is a request and community interest I don't discard this topic could be discussed and visited.

But it's clear that it would require a community traction and involvement.

**Comment by Xunzhuo** [comment]
Thanks forks, maybe we can keep this topic open and keep in touch if I have any progress in chinese localization.
My plan is to write a Chinese handbook about kiali in my own repo first, see https://github.com/Xunzhuo/kiali-handbook, which has not started yet but in my future plan. Hope that I can bring more attention to kiali in China and attract more developers to join into kiali  :) 
 cc @lucasponce  @jshaughn.

**Reply by israel-hdez** [reply]
When it's ready, we could add a link to your repository in the kiali.io website. This would add visibility to it.
